app/controllers/conversations.js

```diff
@@ -29,55 +30,77 @@ class ConversationsController extends BaseController {
     this.sessionRepository = new SessionRepository(ACTIVE);
   }
 
-  async #notifyAboutConversationCreate(
+  async #notifyAboutConversationEvent(
     ws,
     conversation,
-    currentUserLogin,
-    recipients
+    currentUserParams,
+    recipients,
+    eventType
   ) {
     const push_message = {
       title: conversation.name,
-      body: `${currentUserLogin} created a new conversation`,
+      body:
+        getDisplayName(currentUserParams) +
+        CONSTANTS.EVENT_TYPE_PARAMS[eventType].push_message_body,
     };
 
     await PacketProcessor.deliverToUserOrUsers(
       ws,
       {
-        event: { conversation_created: conversation },
+        event: {
+          [CONSTANTS.EVENT_TYPE_PARAMS[eventType].event_request_name]:
+            conversation,
+        },
         push_message,
       },
       conversation._id,
       recipients
     );
   }
 
-  async #notifyAboutConversationUpdate(
+  async #storeMessageAboutParticipantAction(
```

Comment:
let's find a better name for the method. It says `storeMessage`, but in reality the method also sends some data to socket.